### DISCUSSION TITLE
Migrate Data without users

### DISCUSSION THREAD

**Discussion by tw-auh** [netbox-community/netbox#16772] [Help Wanted!]
Hi there,

we've got a Netbox instance hosted by our IT contractor and are not really happy, because they update the server not frequently enough (v3.5 at the moment), so I decided to host it by myself.
The Server is up and running but I'm searching for a way to migrate the data to the new instance.

I have no database access to the old System and I'm afraid that the version there is that old, that a simple dump wouldn't be enough and they will have to upgrade it to a more recent version at first, which comes with a service fee for doing so.
I already tried an CSV export/import but the fields doesn't match so the import failes, I guess it would be the same with the database.

Also, even if I am able to get a database dump and it is importable, I don't want to import the users, because I want to use LDAP in the new instance, since it runs internal.

Any idea what I can do to solve this?

**Comment by markkuleinio** [comment]
As per https://netboxlabs.com/docs/netbox/en/stable/installation/upgrading/ (click the image to show it maybe better), if you get the database dump from 3.5.x, you are able to import that and upgrade to 3.7.x. Then you can upgrade that to 4.x.

You can then delete the users.
